starknet_py/tests/e2e/client/fixtures/transactions.py

```diff
@@ -0,0 +1,30 @@
+from pathlib import Path
+
+import pytest_asyncio
+
+from starknet_py.net import AccountClient
+from starknet_py.net.models.transaction import DeployAccount
+from starknet_py.tests.e2e.conftest import contracts_dir, MAX_FEE
+
+
+@pytest_asyncio.fixture
+async def deploy_account_transaction(
+    new_gateway_account_client: AccountClient,
+) -> DeployAccount:
+    account_client = new_gateway_account_client
+
+    compiled_contract = Path(
+        contracts_dir / "contract_with_validate_deploy.cairo"
+    ).read_text("utf-8")
+    declare_tx = await account_client.sign_declare_transaction(
+        compilation_source=compiled_contract, max_fee=MAX_FEE
+    )
```

Comment:
Are you sure we should name this variable `compiled_contract`? It is not compiled 😕